src/main/client/type.c

```diff
@@ -1065,6 +1066,19 @@ static int AerospikeClient_Type_Init(AerospikeClient *self, PyObject *args,
             &config, strdup((char *)PyUnicode_AsUTF8(py_cluster_name)));
     }
 
+    PyObject *py_app_id = NULL;
+    int retval = PyDict_GetItemStringRef(py_config, "app_id", &py_app_id);
+    if (retval == 1) {
+        const char *str = convert_pyobject_to_str(py_app_id);
+        if (!str) {
+            goto RAISE_EXCEPTION_WITHOUT_AS_ERROR;
```

Comment:
Should py_app_id be decrefed before the goto?

Reply:
Thanks for catching that!